src/SearchComponents/SearchForm.svelte

```diff
@@ -5,37 +5,38 @@
      */
     import SearchField from './SearchField.svelte'
     import {onMount} from 'svelte';
+    import {getLabelForResource, getLabelsForResources, getPropByName} from "../dsp-services";
```

Comment:
_getLabelsForResources_ and _getPropByName_ are not used

Reply:
Removed